src/backend/distributed/executor/adaptive_executor.c

```diff
@@ -2736,7 +2786,7 @@ AvgTaskExecutionTimeApproximation(WorkerPool *workerPool)
 static double
```

Comment:
Not related to this PR but I think we should have the time unit at least in the comment of this function if not in the name